errbot/plugin_manager.py

```diff
@@ -49,35 +49,33 @@ def populate_doc(plugin):
     plugin_type.__errdoc__ = plugin_type.__doc__ if plugin_type.__doc__ else plugin.description
 
 
-def install_package(package):
-    """ Return an exc_info if it fails otherwise None.
+def install_packages(req_path):
+    """ Installs all the packages from the given requirements.txt
+
+        Return an exc_info if it fails otherwise None.
     """
-    log.info("Installing package '%s'." % package)
+    log.info("Installing packages from '%s'." % req_path)
     try:
         if hasattr(sys, 'real_prefix') or (hasattr(sys, 'base_prefix') and (sys.base_prefix != sys.prefix)):
             # this is a virtualenv, so we can use it directly
-            p = subprocess.Popen(['pip', 'install', package])
+            p = subprocess.check_call(['pip', 'install', '-r', req_path])
```

Comment:
It's not a good practice to use short argument flags in scripts, as they're less readable and more likely to change than their long form counterparts. Suggest using `--requirement` instead of `-r`.